src/extension/tags.c

```diff
@@ -493,11 +498,13 @@ static void _dd_http_client_ip(zend_array *meta_ht, zval *_server)
 {
     // If the tracer has set http.client_ip, there is nothing to do here
     // otherwise, this header must be generated here
-    if (zend_hash_exists(meta_ht, _dd_tag_http_client_ip_zstr)) {
+    zval *zv_client_ip = zend_hash_find(meta_ht, _dd_tag_http_client_ip_zstr);
```

Comment:
The tracer doesn't do this anymore although we might add it back again later so it's worth keeping, however I would suggest you check for both the client IP and multiple headers tag in order to avoid recomputing everything.